src/main/java/org/launchcode/liftoffrecipetracker/controllers/CategoryController.java

```diff
@@ -0,0 +1,50 @@
+package org.launchcode.liftoffrecipetracker.controllers;
+
+import org.launchcode.liftoffrecipetracker.data.CategoryRepository;
+import org.launchcode.liftoffrecipetracker.models.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("category")
```

Comment:
I might even change this to categories if this is the path we're going to use to view all, add, edit etc.

Reply:
done!